tests/PubmedIdTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace Altmetric\Identifiers;
+
+class PubmedIdTest extends \PHPUnit_Framework_TestCase
+{
+    public function testExtractsPubMedIDs()
+    {
+        $this->assertEquals(['123', '456', '78910'], PubmedId::extract("123\n456\n78910"));
+    }
+
+    public function testDoesNotReturnOutputsWithPubMedIDsInDOIs()
+    {
+        $this->assertEmpty(PubmedId::extract('10.1038/nplants.2015.3\n10.1126/science.286.5445.1679e'));
+    }
+
+    public function testStripsLeading0s()
+    {
+        $this->assertEquals(['10203', '456000'], PubmedId::extract("0000010203\n000456000"));
```

Comment:
I fear we're being tricked by PHP type juggling here and the IDs are coming back with leading zeroes:

```php
>>> preg_match_all('/(?<=^|\s)0*(?!0)(\d+)(?=$|\s)/', "0000010203\n000456000", $matches)
=> 2
>>> $matches
=> [
     [
       "0000010203",
       "000456000",
     ],
     [
       "10203",
       "456000",
     ],
   ]
>>> "10203" == "0000010203"
=> true
```